lib/chef-dk/skeletons/code_generator/recipes/build_cookbook.rb

```diff
@@ -3,18 +3,22 @@
 delivery_project_dir = context.delivery_project_dir
 pipeline = context.pipeline
 dot_delivery_dir = File.join(delivery_project_dir, ".delivery")
+config_json = File.join(dot_delivery_dir, "config.json")
+project_toml = File.join(dot_delivery_dir, "project.toml")
 
 generator_desc("Ensuring delivery configuration")
 
 directory dot_delivery_dir
 
-cookbook_file File.join(dot_delivery_dir, "config.json") do
+cookbook_file config_json do
   source "delivery-config.json"
+  not_if {File.exist?(config_json)}
```

Comment:
nit: property style here is `not_if { File.exist?(config_json) }` (mostly for readability)